_articles/disaster-recovery.md

```diff
@@ -46,7 +46,9 @@ Once the drive is mounted, we can install software to back up files.
 
 [Déjà Dup](https://wiki.gnome.org/Apps/DejaDup) is a popular, straightforward backup option. It can be used to back up user data locally, remotely, or to a cloud server. It can also be set up to run automatically on a regular basis.
 
-[rsync]() is a popular command-line tool to copy data while keeping the permissions. It is useful if the system is powering down or if you want a command-line tool to copy the files. 
+### Rsync
+
+[rsync](https://www.digitalocean.com/community/tutorials/how-to-use-rsync-to-sync-local-and-remote-directories) is a popular command-line tool to copy data while keeping the permissions. It is useful if the system is powering down or if you want a command-line tool to copy the files. 
```

Comment:
Please note the cloud storage policy may affect upload/download speed.  This was an issue at an old Job @zachwingrave and I had, the uploads from Linux were snail-paced.

Reply:
Added a note related to this.